.github/scripts/releasecli.sh

```diff
@@ -1,9 +1,10 @@
+GITHUB_SHA=$1
 VERSION=$(cat version)
 BASE_URL="https://console.cloud.google.com/storage/browser/inspr-cli/bin"
 
 release-cli create --name "Release $VERSION" \
         --tag-name $VERSION \
-        --ref "$CI_COMMIT_SHA" \
+        --ref GITHUB_SHA \
```

Comment:
Same as before, does this actually work without the `$` before the `GITHUB_SHA`.

i tested doing something similar in my local machine and the value stored in the variable is only used when adding the `$`.